### PR TITLE
[filter-fdb] Fix For Vlan Defined With No CIDR

### PR DESCRIPTION
**Pull request by tahmed-dev**
VLAN_INTERFACE section in Config_db may contain entries with now CIDR.
This fix skip those entries.

signed-off-by: Tamer Ahmed <tamer.ahmed@microsoft.com>

Fixes #4933

**- What I did**
Fix for VLAN_INTERFACE with no CIDR

**- How I did it**
Skipped over entries in VLAN_INTERFACE section that do not have CIDR

**- How to verify it**
***- Without This Fix***
```
taahme@024d65787c01:/sonic/sonic-utilities$ sudo pytest sonic-utilities-tests/filter_fdb_entries_test.py  -vvvv
=================================================================================================================================================== test session starts ===================================================================================================================================================
platform linux2 -- Python 2.7.13, pytest-3.0.6, py-1.4.32, pluggy-0.4.0 -- /usr/bin/python
cachedir: sonic-utilities-tests/.cache
rootdir: /sonic/sonic-utilities/sonic-utilities-tests, inifile: pytest.ini
plugins: cov-2.4.0
collected 10 items 

sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData0] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData1] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData2] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData3] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData4] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData5] FAILED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData6] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData7] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData8] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData9] PASSED

======================================================================================================================================================== FAILURES =========================================================================================================================================================
```
***- With This Fix***
```
taahme@024d65787c01:/sonic/sonic-utilities$ sudo pytest sonic-utilities-tests/filter_fdb_entries_test.py  -vvvv
=================================================================================================================================================== test session starts ===================================================================================================================================================
platform linux2 -- Python 2.7.13, pytest-3.0.6, py-1.4.32, pluggy-0.4.0 -- /usr/bin/python
cachedir: sonic-utilities-tests/.cache
rootdir: /sonic/sonic-utilities/sonic-utilities-tests, inifile: pytest.ini
plugins: cov-2.4.0
collected 10 items 

sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData0] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData1] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData2] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData3] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData4] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData5] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData6] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData7] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData8] PASSED
sonic-utilities-tests/filter_fdb_entries_test.py::TestFilterFdbEntries::testFilterFdbEntries[testData9] PASSED

================================================================================================================================================ 10 passed in 0.34 seconds ================================================================================================================================================
```
**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: scripts/filter_fdb_entries.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ def get_vlan_cidr_map(filename):
     vlan_cidr = defaultdict()
     if "VLAN_INTERFACE" in config_db_entries.keys() and "VLAN" in config_db_entries.keys():
         for vlan_key in config_db_entries["VLAN_INTERFACE"].keys():
+            if '|' not in vlan_key:
```
Comment (prsunny) [inline scripts/filter_fdb_entries.py:34]: Can you please use this [method ](https://github.com/Azure/sonic-utilities/blob/master/scripts/db_migrator.py#L64)to check this for consistency?

Comment (tahmed-dev) [reply scripts/filter_fdb_entries.py:34]: how does this work? the key is a string, how would it be a tuple?

Comment (prsunny) [reply scripts/filter_fdb_entries.py:34]: well, if it has the ip prefix, it is a tuple.. if not, its a string.

Comment (tahmed-dev) [reply scripts/filter_fdb_entries.py:34]: has an offline chat with @prsunny regarding this. This does not work here as the referenced call had an underlying API that would convert the key into tuple. In this case on hand, the processing is done within this script.

Comment (prsunny) [reply scripts/filter_fdb_entries.py:34]: is it better then to check for "/" to check if prefix is present?

**Pull request merged**